### ISSUE TITLE
V3: Studios/Performers >100 Scenes Do Not Download Full List

### ISSUE THREAD

**Issue by Eleniatari** [Whisparr/Whisparr#187]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

When I monitor and refresh a site, either manually or waiting for an automatic refresh if the performer or studio has >100 scenes not all will download. There is a high prevalence to stopping at multiples of 100's of scenes.

I only have 14k scenes monitored or unmonitored in whisparr.

### Expected Behavior

It is expected for all scenes to become available.

### Steps To Reproduce

Sites/Studios with <100 scenes work fine.
All that is required to reproduce is finding a studio or performer with >100 scenes, monitor and refresh then check the available in Whisparr vs Stash.

Studio: BFFs
Whisparr: 200
Stash: 231

Studio: Hentaied
Whisparr: 106
Stash: 171

Studio: TeamSkeet Classics
Whisparr: 100
Stash: 105

_Studio: TeamSkeet Labs
Whisparr: 99
Stash: 99_

Performer: Whitney Wright
Whisparr: 600
Stash 635

Performer: Eva Elfie
Whisparr: 197
Stash: 221

Every one I checked has a discrepancy if >100 scenes on stash.

### Environment

```markdown
- OS: Synology
- Whisparr: 3.0.0.503
- Docker Install: Yes
- Using Reverse Proxy: No
- Browser: Firefox
- Database:
```

### What branch are you running?

Develop

### Trace Logs?

[whisparr.trace.2.txt](https://github.com/Whisparr/Whisparr/files/14670388/whisparr.trace.2.txt)
Here is the entire trace for attempting to update Eva Elfie

**Comment by plz12345** [comment]
I confirmed same, and it doesn't look to be Whisparr's fault.  SkyHook data discrepancy with StashDB appears likely.

**Skyhook** `GET https://api.whisparr.com/v4/performer/6f602bca-36c2-40ed-b1bb-0aad4768b57f/works`
Scene count: 200

**StashDB** `POST https://stashdb.org/graphql`
**Payload**
``` json
{"operationName":"Scenes","variables":{"input":{"page":1,"per_page":40,"sort":"DATE","direction":"DESC","performers":{"value":["6f602bca-36c2-40ed-b1bb-0aad4768b57f"],"modifier":"INCLUDES"},"studios":{"value":[],"modifier":"INCLUDES"}}},"query":"query Scenes($input: SceneQueryInput!) {\n  queryScenes(input: $input) {\n    count\n    scenes {\n      ...QuerySceneFragment\n      __typename\n    }\n    __typename\n  }\n}\n\nfragment URLFragment on URL {\n  url\n  site {\n    id\n    name\n    icon\n    __typename\n  }\n  __typename\n}\n\nfragment ImageFragment on Image {\n  id\n  url\n  width\n  height\n  __typename\n}\n\nfragment ScenePerformerFragment on Performer {\n  id\n  name\n  disambiguation\n  deleted\n  gender\n  aliases\n  __typename\n}\n\nfragment QuerySceneFragment on Scene {\n  id\n  release_date\n  title\n  duration\n  urls {\n    ...URLFragment\n    __typename\n  }\n  images {\n    ...ImageFragment\n    __typename\n  }\n  studio {\n    id\n    name\n    __typename\n  }\n  performers {\n    as\n    performer {\n      ...ScenePerformerFragment\n      __typename\n    }\n    __typename\n  }\n  __typename\n}"}
```
Scene count: 224

**Comment by sampulsar** [comment]
Looked at the log on the ticket and the skyhook server failed to respond with the scene data. As it is returning a 404 with no data, as if the server does not have the data.

There is one that still fails, https://api.whisparr.com/v4/scene/4a89a0e3-831d-4d7b-8cc7-4abb857de3c3: 404.NotFound (0 bytes) (27 ms), it has 2023 as the date. Some others are working correctly now, but failed at the time.
https://api.whisparr.com/v4/scene/30bb02cc-c3ea-4cea-8a1b-db41846d3954: 404.NotFound (0 bytes) (36 ms)

Something internal with Skyhook, could be a rate limit or bad data at the time. You would assume that the Skyhook server would be logging any errors. If it is a rate limit then Whisparr could be altered to grab the data slower.

Could add the logging to see how many scenes are in the performers works, the number expected to be added and then the number actually added. This can then confirm if the errors due to skyhook not returning the data is the only issue.

**Comment by plz12345** [comment]
I don't believe this is still an issue.  Tested by adding multiple studios with 500+ scenes.  If it is, please reply.

**Comment by spanks106** [comment]
This is still an issue, mypervyfamily 797 on stash db I have whisparr only pulling 22 episodes

**Comment by plz12345** [comment]
Open a new issue, this was closed two months ago.  We'll need logs on the new issue you open.
